CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+## Unreleased
```

Comment:
Don't edit CHANGELOG.md in PRs, our workflow is the edit CHANGELOG_PENDING.md instead.

Reply:
My bad, fixed!